src/Repository/InstalledBootstrap.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace Phpcq\Repository;
+
+use Phpcq\Exception\RuntimeException;
+
+/**
+ * Locally installed bootstrap.
+ */
+class InstalledBootstrap implements BootstrapInterface
+{
+    /**
+     * @var string
+     */
+    private $filePath;
+
+    public function __construct(string $version, string $filePath)
+    {
+        if ($version !== '1.0.0') {
```

Comment:
How will we managed potential upcoming versions? Should we think about it now or if the case happens?

Reply:
I think there should be no problems. We could either instantiate a different class then (and still have this same check in place) or handle the new version differently in this class.
Therefore we should be safe to post pone this discussion.